aten/src/ATen/native/README.md

```diff
@@ -170,6 +170,27 @@ Note that argument type modifiers such as defaults and optional are not currentl
 
 The declarations also support the following attributes:
 
+**Overloads.** You can register multiple functions with the same name and different
+function signatures if you give them unique overload names. An overload name
+is specified after the function name, separated by a dot.
+
+Overload names do not have to be globally unique, but must be unique in the set
+of all overloads for the same function. Please try to make overload names
+semantically meaningful. An overload name that just enumerates the argument
+types isn't helpful. Actually it is hurtful, because overload names cannot be
```

Comment:
I don't get it.  The above hints that I need to give an overload name ("You can register multiple functions with the same name and different function signatures if you give them unique overload names") -- okay technically it doesn't say if and only if...

But then here it tells me if I give it a bad name it's hurtful?  But how could it be worse than something that doesn't work?